factory/usr/lib/the-tool

```diff
@@ -4,14 +4,3 @@ set -e
 # Run snap-bootstrap
 /usr/lib/snapd/snap-bootstrap initramfs-mounts
 
-# TODO:UC20: move this code into snap-bootstrap proper so this script _just_ 
```

Comment:
this doesn't have to be done as part of this PR and in fact can be done after your chain of PR's have all been landed, but it would be nice to get rid of this wrapper script as mentioned

Reply:
Yes, I have noted that. I will come back to it. This is just not needed for stateful re-execution so I will do it after.